apps/embed-element-app/src/App.tsx

```diff
@@ -35,32 +36,31 @@ const events = [
 ];
 
 function App() {
-    const roomUrl = "enter your room url here";
-    const elmRef = useRef<HTMLIFrameElement>(null);
+    const roomUrl ="enter your room url here";
+    const elmRef = useRef<WherebyEmbedElement>(null);
```

Comment:
It's failing because of this line. We might have an error in the type of this element. Maybe just switch it back for now?

Reply:
Boy oh boy 🤦‍♂️. What a day for Nick Rolls 😂. fixed